doc/concepts/labels.rst

```diff
@@ -293,6 +293,36 @@ Locset expressions
       On the left is the region with radius between 0.3 μm and 0.5 μm.
       The right shows the proximal set of this region.
 
+.. label:: (proximal-translate ls:locset distance:real)
+
+    The set of locations that correspond to moving each location in the ``ls`` in the proximal direction
+    ``distance`` μm. The locations in the output have a one to one correspondance with those in ``ls``.
+
+    .. figure:: ../gen-images/proximal_translate_label.svg
+      :width: 600
+      :align: center
+
+      The proximal translation of the terminal locations (left) a distance of 10 μm using the
+      expression ``(proximal-translate (terminal) 10)``.
+
+.. label:: (distal-translate ls:locset distance:real)
+
+    The set of locations that correspond to moving each location in ``ls`` in the distal direction
+    ``distance`` μm or to a terminal location, whichever is closest.
+    The number of locations in the output is greater than or equal the number in the input, with
+    multiple locations generated from input locations that are less than ``distance`` μm
```

Comment:
more than `distance`?

Reply:
I think I mean less, but might have to reword it to make sense.

Reply:
Ah I actually see it now.